agenta-web/src/pages/apps/[app_id]/overview/index.tsx

```diff
@@ -18,6 +16,8 @@ import {Button, Dropdown, Space, Typography} from "antd"
 import {useRouter} from "next/router"
 import {useCallback, useEffect, useState} from "react"
 import {createUseStyles} from "react-jss"
+import DeleteAppModal from "@/components/pages/app-management/modals/DeleteAppModal"
```

Comment:
might be a better practice to load modals dynamically since they are not shown directly when landing on the page